app/helpers/react_components/mentoring/session/exemplar_file_list.rb

```diff
@@ -0,0 +1,23 @@
+module ReactComponents
```

Comment:
I didn't want to add this to `serializers/` because it's been getting harder for me to navigate that directory. I think we ought to have some form of object that is local to the react component it is for. I'll have a play with this approach and see where it takes us.

Reply:
Yep. Serializers should only be used for things that use the API. Other things should be inlined like you've done here.

Reply:
@kntsoriano I've inlined this into the React component. We don't need extra files for every helper class that's used - we can add things into the one class instead. This is how Rails organises everything internally and it means you don't need to jump around a code base to understand what's going on, which I much prefer.

Reply:
My reason with breaking this up was to be able to test only what we've changed. It's getting frustrating that when we change one thing here, we need to fix three other tests. I think this is a symptom of not testing things in the right level.

Reply:
Yeah - totally agree with this too. We should only need to change 1 test if this changes. See https://github.com/exercism/website/pull/1975